app/config/auth.js

```diff
@@ -18,7 +17,6 @@ export const getAuthConfig = () => {
       hostname: `https://${process.env.DEFRA_ID_TENANT}.b2clogin.com/${process.env.DEFRA_ID_TENANT}.onmicrosoft.com`,
       oAuthAuthorisePath: '/oauth2/v2.0/authorize',
       policy: process.env.DEFRA_ID_POLICY,
-      dashboardRedirectUri: process.env.DEFRA_ID_DASHBOARD_REDIRECT_URI,
       clientId: process.env.DEFRA_ID_CLIENT_ID,
```

Comment:
Quite a few other config items in here that could probably go now for Defra id, APIM, RPA etc

Reply:
Yeah, realised we can just remove the whole auth config from this repo